site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -146,8 +147,15 @@ They can include JavaScript files, CSS files, or both. All auto-run content scri
       <td>
         <nobr>array of strings</nobr>
       </td>
-      <td><em>Optional.</em> The list of JavaScript files to be injected into matching pages. These
-        are injected in the order they appear in this array.</td>
+      <td><em>Optional.</em> The list of JavaScript files to be injected into matching pages. Files
+        are injected in the order they appear in this array. Each string in this list must contain
+        the either an absolute or relative path from the extension's root directory.</td>
```

Comment:
There's no such thing as an absolute path within a directory - we just helpfully strip leading '/' characters.

Let's just say "A relative path to a resource in the extension's root directory"

Reply:
Ack. I've also included a note about removing the leading slash as I have been a little confused by that behavior in the past.

Reply:
Ack. I've also included a note about removing the leading slash as I have been a little confused by that behavior in the past.